### ISSUE TITLE
Keyborad input didn't react.

### ISSUE THREAD

**Issue by HEKEPOIU** [ntut-open-source-club/practical-tools-for-simple-design#128]
### Problem

In app.cpp 
```cpp
    if (Util::Input::IsKeyDown(Util::Keycode::A)) {
        LOG_DEBUG("A Down");
    }
```
I press a and didn't react.

### Steps to reproduce

- clone project 
- build project
- run and press A

### Expected behavior

log "A Press"

### PTSD version

873d464f147bac9e9e05e693f1570b1003f67029

### Operating system/version

ubuntu 22.04.3

### CMake version

3.28.3

### CMakeCache.txt

[CMakeCache.txt](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/files/14372180/CMakeCache.txt)

### OpenGL Info

```txt
[info] OpenGL Info
 [info]   Vendor: NVIDIA Corporation
 [info]   Renderer: NVIDIA GeForce RTX 3060 Laptop GPU/PCIe/SSE2
 [info]   Version: 4.1.0 NVIDIA 535.154.05
 [info]   GLSL Version: 4.10 NVIDIA via Cg compiler
```

**Comment by NOOBDY** [comment]
is your log level correct?

**Comment by HEKEPOIU** [comment]
How to check?

**Comment by NOOBDY** [comment]
you can set it at `include/config.hpp:16`

**Comment by Dragonfly911117** [comment]
> is your log level correct?

I think the default level is `DEBUG`, which should enable the log.

**Comment by Dragonfly911117** [comment]
However, I was not able to reproduce the issue on my Arch machine with Sway as WM. Can you further provide your WM/DM?

**Comment by HEKEPOIU** [comment]
yes it is ```constexpr Util::Logger::Level DEFAULT_LOG_LEVEL = Util::Logger::Level::DEBUG;```
and mouse input part log correctly.

**Comment by HEKEPOIU** [comment]
mnn? how to get WM/DM?

**Comment by Dragonfly911117** [comment]
Not sure if [this](https://askubuntu.com/questions/72549/how-to-determine-which-window-manager-and-desktop-environment-is-running) works.
PS `neofetch` may simply do the job.

**Comment by HEKEPOIU** [comment]
![image](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/assets/101374163/2d66403a-460d-4628-a6fd-98f0da191a20)
(this?)
and i found that i press "ctrl+a" it will trigger.
some time can trigger normaly, but some time need key + "ctrl"

**Comment by ntut-rick** [comment]
> i found that i press "ctrl+a" it will trigger.

same issue on my windows 10.
pressing the `A` alone doesn't trigger `Util::Input::IsKeyDown(Util::Keycode::A)`, but pressing `Ctrl+A` does

**Comment by ntut-rick** [comment]
can you check if this issue related to #65 ?
